lib/compat/wordpress-6.0/script-loader.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Load the user preset styles separately with lower priority to they will be more
+ * likely to load after any theme css that they need to override.
+ */
+
+function gutenberg_enqueue_user_preset_styles() {
+	$presets_stylesheet = gutenberg_get_global_stylesheet( array( 'presets' ) );
+
+	wp_register_style( 'use-preset-styles', false, array(), true, true );
+	wp_add_inline_style( 'use-preset-styles', '.has-background { background-color: var(--wp--user--preset--background-color);} .has-text-color {color: var(--wp--user--preset--color);}' . $presets_stylesheet );
+	wp_enqueue_style( 'use-preset-styles' );
+}
+add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_user_preset_styles', 100 );
```

Comment:
If the goal is to prevent overriding the preset classes defined by a theme (if they enqueue any), why do we try to load the core preset classes _after_?

Reply:
It's so they come later in the cascade and win specificity over any previous styles without using `!important` or an ID selector. This is common practice for using utility-style CSS alongside traditional styles: [Bootstrap](https://github.com/twbs/bootstrap/blob/main/scss/bootstrap.scss), [ITCSS](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/)